dev/src/Presentation/MainMenuScreen.java

```diff
@@ -13,10 +13,11 @@ public MainMenuScreen(PrintStream out, Scanner in) {
     public int handleMsg() {
         this.out.println("Main Menu");
 
-        this.out.println("1. To create new store");
-        this.out.println("2. Suppliers");
-        this.out.println("3. Inventory");
-        this.out.println("4. Exit");
+        this.out.println("1. Add Store");
+        this.out.println("2. Display Store");
+        this.out.println("3. Add Supplier");
+        this.out.println("4. Display Supplier");
```

Comment:
שולי אבל ספקים וחנויות ברבים ולא יחיד display stores display supliers

Reply:
Fixed in #50